docs/source/fabric-tutorial/index.md

```diff
@@ -0,0 +1,22 @@
+These tutorials are specifically aimed at using Caliper to performance test Hyperledger Fabric networks.
+
+<style>
+.clickable-rectangle {
+    display: block;
+    width: 100%;
+    background-color: #c8d3d5;
+    color: black !important;
+    padding: 20px;
+    text-decoration: none;
+    font-size: 16px;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    transition: background-color 0.3s;
+}
+
+</style>
+
+<a href="https://hyperledger.github.io/caliper/v0.6.0/fabric-tutorial/tutorials-fabric-existing/" class="clickable-rectangle">

```

Comment:
If I click the button, it takes me to the current published caliper fabric tutorial and not the tutorial in the docs). However I've never liked this button approach because it's not obvious that something needs to be clicked here. Not sure what makes sense but it should be obvious that it can be clicked and that it takes you to the correct place.

Reply:
Can we directly have the content?

Reply:
The content is already there in the fabric-tutorial folder. Given that I hope you will be adding more tutorials we should change that folder name from `fabric-tutorial` to `tutorials` and we could even change the name of the sub folder from `tutorials-fabric-existing` to `fabric`